models/snapchat_ads__campaign_country_report.sql

```diff
@@ -0,0 +1,69 @@
+{{ config(enabled=var('ad_reporting__snapchat_ads_enabled', true) and var('snapchat__using_campaign_country_report', false)) }}
+
+with campaign_daily as (
+
+    select *,
+        {% if var('snapchat_ads__conversion_fields', none) %}
+            {{ var('snapchat_ads__conversion_fields') | join(' + ') }} as total_conversions
+        {% else %}
+            0 as total_conversions
+        {% endif %}
+    from {{ var('campaign_country_daily_report') }}
+
+), account as (
+
+    select *
+    from {{ var('ad_account_history') }}
+    where is_most_recent_record = true
+
+), campaigns as (
+
+    select *
+    from {{ var('campaign_history') }}
+    where is_most_recent_record = true
+
+
+), aggregated as (
+
+    select
+        campaign_daily.source_relation,
+        campaign_daily.date_day,
+        campaign_daily.campaign_id,
+        campaigns.campaign_name,
+        campaign_daily.country,
+        account.ad_account_id,
+        account.ad_account_name,
+        campaigns.created_at,
+        campaigns.daily_budget,
+        campaigns.start_time,
+        campaigns.end_time,
+        campaigns.status,
+        campaigns.objective,
+        campaigns.lifetime_spend_cap,
+        account.currency,
```

Comment:
Could you alias these as `campaign_<field>` or `account_<field>` to clarify what they refer to?